cabal.project.local

```diff
@@ -0,0 +1 @@
+ignore-project: False
```

Comment:
Probably should be part of gitignore ? Same with the binary files checked in `dist-newstyle/cache/config` etc.